docs/it-manual/sre-playbook/monitoring.md

```diff
@@ -57,3 +60,68 @@ To import this pre-built dashboard:
 5. Paste [the JSON here](https://gist.githubusercontent.com/dkatzz/a7048a6db9a2cbb14ef96f744f5fd964/raw/f65295421e47782cc55eecdbf70653d676121070/grafana_json) into the "Import via panel JSON" 
 7. Click "Load"
 8. Fill in the details for the imported dashboard, selecting your CiviForm prometheus instance for the data source
+
+## Additional AWS monitoring
+
+In addition to the server metrics provided by Prometheus, there are some additional places within AWS you can go to see metrics.
+
+### CloudWatch
+
+#### Dashboards
+
+CloudWatch has some default dashboards that allow you to see graphs with metrics on different parts of the deployment. Not all of these are relevant, but these can be helpful in seeing CPU utilization in RDS (the database) and Elastic Container Service, as well as requests to the Application ELB (load balancer), metrics about S3, etc.
+![](<../../.gitbook/assets/aws-cloudwatch-dashboards.png>)
```

Comment:
**nitpick:** Some feedback I got on the existing docs is that they're not quite as beginner friendly as they could be, so these suggestions are mostly in service of remedying that.
```suggestion
[CloudWatch](https://aws.amazon.com/cloudwatch/) has some default dashboards that allow you to see graphs with metrics on different parts of the deployment. Not all of these are relevant, but these can be helpful in seeing CPU utilization in [RDS](https://aws.amazon.com/rds/) (CiviForm's PostgreSQL database) and [ECS](https://aws.amazon.com/ecs/) (server hosting), as well as requests to the [ALB](https://aws.amazon.com/elasticloadbalancing/application-load-balancer/) (load balancer), metrics about [S3](https://aws.amazon.com/s3/) (file storage), etc.
![](<../../.gitbook/assets/aws-cloudwatch-dashboards.png>)
```